<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shuju</class>
 <widget class="QWidget" name="shuju">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="txt_role">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_id">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>395</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnquery">
      <property name="text">
       <string>查询内容</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btninsert">
      <property name="text">
       <string>输入内容</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btndelete">
      <property name="text">
       <string>删除数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnupdate">
      <property name="text">
       <string>更新数据</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="txt_pwd">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>861</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
